apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -86,24 +83,22 @@ function TokenDetails({ coinType }: TokenDetailsProps) {
                 className="flex flex-col h-full flex-1 flex-grow items-center"
                 data-testid="coin-page"
             >
+                {!coinType && <AccountSelector />}
+                <div className="mt-1.5">
+                    <CoinBalance
```

Comment:
if we don't include the loading state here we probably will show 0 and then the actual balance. Why do we have to remove the loading state?

Reply:
If fetching the data fails it keeps switching flicking, this just makes sure that it shows the 0 and the error message

Reply:
With the side effect that we show 0 even when we are loading the balance. I think we should find a better way to fix this.

Reply:
Adding `const isFirstTimeLoading = isLoading && !isFetched;`